build_extra/rust/rust.gni

```diff
@@ -1,4 +1,12 @@
-stdlib_label = "//build_extra/rust:stdlib"
+empty_rs_path = rebase_path("empty.rs", root_build_dir)
+rust_bin_ldflags =
+    exec_script("get_rust_ldflags.py", [ empty_rs_path ], "list lines")
+rust_test_ldflags = exec_script("get_rust_ldflags.py",
+                                [
+                                  empty_rs_path,
+                                  "--test",
+                                ],
+                                "list lines")
```

Comment:
list lines?

Please point the reader to the soon-to-be-written comment at the top of get_rust_ldflags.py

Reply:
gn help exec_script